app/controllers/SeriesDetailsController.scala

```diff
@@ -97,6 +95,12 @@ class SeriesDetailsController @Inject() (
     }
     DropdownField(form("series").id, "", "", "", Nil, multiValue = false, options, None, isRequired = true, errors.toList)
   }
+
+  def addSeriesStatus(seriesStatus: Option[String], consignmentId: UUID, request: Request[AnyContent]) = {
```

Comment:
Sorry, just a small thing, actually, if we are not using any method outside of the class then we should make it private but I think you can still test it with the main method.

Reply:
No worries. I've moved the method back into the main method and made changes to the tests.